scripts/varsome_api_annotate_vcf.py

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env python
```

Comment:
Many (all?) of these files are missing (c) 2018 Saphetor at the top.
We need to decide which license we are using for these.
And then include the appropriate header in all files.
Preferably something that means that:
1- We own the original code.
2- If somebody modifies the code they still have to include the copyright message.
3- Would be nice if they send bug-fixes back to us.
4- They can sell the code to 3rd party uses, but they must credit Saphetor.

Reply:
Not sure if we need all this info in all files. The repository contains a license file which can cover all the code

Reply:
e.g. https://github.com/elastic/elasticsearch-py/blob/master/LICENSE